utils/test-utils/src/main/java/com/microsoft/device/dualscreen/testing/DeviceModel.kt

```diff
@@ -109,5 +118,9 @@ private fun UiDevice.getModelFromPaneWidth(paneWidth: Int): DeviceModel {
         "DeviceModel",
         "Unknown dualscreen device dimensions $displayWidth $displayHeight"
     )
-    return DeviceModel.Other
+    return DeviceModel.Other.apply {
+        this.paneWidth_ = displayWidth / 2
+        this.paneHeight_ = displayHeight
+        this.totalDisplay_ = paneWidth_ * 2 + foldWidth
+    }
```

Comment:
This is a great addition, good idea! We didn't end up adding other foldable models to `DeviceModel` for the first version of the Test Kit, but if you feel that more models would be useful (especially for the @TargetDevice annotation), I think you could definitely add more models to the class

Reply:
Yeah. In the future I want to add enums for 6.7", 7.6", 8" devices. Default that two panes will be equal. I can't retrieve this information about UIDevice or WindowManager panels synchronously, so we'll assume they're equal.

Reply:
Awesome, and yes that makes sense, it's a great idea to note that in the documentation 😄